packages/@aws-cdk/aws-cognito/test/user-pool-client.test.ts

```diff
@@ -283,4 +283,40 @@ describe('User Pool Client', () => {
       AllowedOAuthScopes: [ 'aws.cognito.signin.user.admin' ],
     });
   });
+
```

Comment:
Add another test case that `PreventUserExistenceError` is `ABSENT` when undefined.